gammapy/scripts/image_pipe.py

```diff
@@ -325,3 +384,35 @@ def excess_image(self):
         total_excess = SkyMap.empty_like(self.empty_image)
         total_excess.data = self.maps["counts"].data - self.maps["bkg"].data
         self.maps["excess"] = total_excess
+
+    def make_mean_psf_tab(self, region_center, spectral_index=2.3):
```

Comment:
This is the example we wrote this morning how I would have implemented the mean PSF computation:
https://gist.github.com/cdeil/8f61fc0cb0604c9115942da283da9552

As we said, maybe this methods should be moved to a `DataStoreObservation` method.
And the mean PSF computation across observations should be separate from the integration of the PSF for a given energy band (that's the last step, a spectral model is only needed there).

Can you add an example or test to this PR, so that I can try out this code.